way_to_home/place/models.py

```diff
@@ -1 +1,68 @@
 """This module implements class that represents the place entity."""
+
+from user.models import User
+from django.db import models, IntegrityError
+from utils.abstract_models import AbstractModel
+
+
+class Place(AbstractModel):
+    """Model for place entity."""
+
+    longitude = models.DecimalField()
+    latitude = models.DecimalField()
+    name = models.CharField(max_length=64)
+    address = models.CharField(max_length=255)
+    stop_id = models.PositiveSmallIntegerField()
```

Comment:
```suggestion
    stop_id = models.PositiveSmallIntegerField()
```
stop_id can be empty. set `null=True`

Reply:
Done